testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -130,29 +130,44 @@ impl SignerTest<SpawnedSigner> {
 
         info!("Got {} signatures", signature.len());
 
-        assert_eq!(signature.len(), num_signers);
+        // NOTE: signature.len() does not need to equal signers.len(); the stacks miner can finish the block
+        //  whenever it has crossed the threshold.
+        assert!(signature.len() >= num_signers / 7 * 10);
```

Comment:
Did you mean to write

```suggestion
        assert!(signature.len() >= num_signers * 7 / 10);
```

Reply:
Yes, good catch!

Reply:
haha that is definitely what I meant to do XD